Core/Core/Credentials/AccountManager.cs

```diff
@@ -107,7 +107,7 @@ private static async Task<UserServerInfoResponse> GetUserServerInfo(string token
       var request = new GraphQLRequest
       {
         Query =
-          @"query { user { id name email company avatar streams { totalCount } commits { totalCount } } serverInfo { name company adminContact description version} }"
+          @"query { activeUser { id name email company avatar streams { totalCount } commits { totalCount } } serverInfo { name company adminContact description version} }"
```

Comment:
This is fine, but we also have to modify (or duplicate+obsolete) the `UserServerInfoResponse` so that it contains an `activeUser` field instead of a `user` field, or the query will succeed but the user will be null.